apps/vs-code-designer/src/app/commands/createNewCodeProject/cloudToLocal.ts

```diff
@@ -21,14 +23,99 @@ import { setWorkspaceName } from './CodeProjectBase/SetWorkspaceName';
 import { AzureWizard } from '@microsoft/vscode-azext-utils';
 import type { IActionContext } from '@microsoft/vscode-azext-utils';
 import { latestGAVersion, OpenBehavior } from '@microsoft/vscode-extension-logic-apps';
-import type { ICreateFunctionOptions, IFunctionWizardContext, ProjectLanguage } from '@microsoft/vscode-extension-logic-apps';
+import type {
+  ConnectionReferenceModel,
+  ICreateFunctionOptions,
+  IFunctionWizardContext,
+  ProjectLanguage,
+} from '@microsoft/vscode-extension-logic-apps';
 import { window } from 'vscode';
 import * as path from 'path';
-import * as util from 'util';
 import AdmZip = require('adm-zip');
 import * as fs from 'fs';
+
+// Constants
 const openFolder = true;
+const DELIMITER = '/';
+const SUBSCRIPTION_INDEX = 2;
+const MANAGED_API_LOCATION_INDEX = 6;
+const MANAGED_CONNECTION_RESOURCE_GROUP_INDEX = 4;
+
+// Function to create an AdmZip instance
+function createAdmZipInstance(zipFilePath: string) {
+  return new AdmZip(zipFilePath);
+}
+
+// Function to get zip entries
+function getZipEntries(zipFilePath: string) {
+  const zip = createAdmZipInstance(zipFilePath);
+  return zip.getEntries();
+}
+
+// Function to deep merge objects
+function deepMergeObjects(target: any, source: any): any {
```

Comment:
I think there is a `deepMerge` function in libs/logic-apps-shared/src/utils/src/lib/helpers/functions.ts Lets try to reuse it. Lets see if it works for your scenario, you just need to export it and import in in here

Reply:
Fixed this mistake with the new commit